### ISSUE TITLE
Server Error (500) from `/topic/{id}/credentialset`

### ISSUE THREAD

**Issue by WadeBarnes** [bcgov/aries-vcr#486]
Found in:
- 87c6bd423f537231d36ffa8ae8c26f91247f3b9f

Steps to reproduce:
- The issue occurs for this corp; https://orgbook-dev.pathfinder.gov.bc.ca/en/organization/registration.registries.ca/BC0131809 
- Specifically for this API call; https://orgbook-dev.pathfinder.gov.bc.ca/api/topic/170966/credentialset

Expected result:
- The error to be handled gracefully, or not happen at all.

Observed Result:
![image](https://user-images.githubusercontent.com/10504350/80502323-56b94880-8925-11ea-894b-a4434b60ba9a.png)

Error Log:
```
ERROR 2020-04-24 21:16:51,455 log 1 140514946967296 Internal Server Error: /api/topic/170966/credentialset
Traceback (most recent call last):
  File "/home/indy/.pyenv/versions/3.6.8/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/indy/.pyenv/versions/3.6.8/lib/python3.6/site-packages/django/core/handlers/base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/indy/.pyenv/versions/3.6.8/lib/python3.6/site-packages/django/core/handlers/base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/indy/.pyenv/versions/3.6.8/lib/python3.6/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/indy/.pyenv/versions/3.6.8/lib/python3.6/site-packages/rest_framework/viewsets.py", line 114, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/indy/.pyenv/versions/3.6.8/lib/python3.6/site-packages/rest_framework/views.py", line 505, in dispatch
    response = self.handle_exception(exc)
  File "/home/indy/.pyenv/versions/3.6.8/lib/python3.6/site-packages/rest_framework/views.py", line 465, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/indy/.pyenv/versions/3.6.8/lib/python3.6/site-packages/rest_framework/views.py", line 476, in raise_uncaught_exception
    raise exc
  File "/home/indy/.pyenv/versions/3.6.8/lib/python3.6/site-packages/rest_framework/views.py", line 502, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/indy/api/v2/views/rest.py", line 271, in list_credential_sets
    for credential_set in credential_sets
  File "/home/indy/api/v2/views/rest.py", line 271, in <listcomp>
    for credential_set in credential_sets
  File "/home/indy/api/v2/views/rest.py", line 268, in <listcomp>
    for credential in credential_set.credentials.all()
  File "/home/indy/api/v2/views/rest.py", line 261, in <listcomp>
    for related_topic in credential.related_topics.all()
AttributeError: 'NoneType' object has no attribute 'id'
```

**Comment by WadeBarnes** [comment]
Another example of this issue can be found here:
- https://orgbook-test.pathfinder.gov.bc.ca/en/organization/registration.registries.ca/BC0499790

**Comment by ianco** [comment]
Company BC0131809

**Comment by ianco** [comment]
Seems to be missing FM0149073 in this environment

**Comment by WadeBarnes** [comment]
> Seems to be missing FM0149073 in this environment

The `dev` environment is only a partial data load.  So that's not completely surprising.  The ones from `test` will be interesting to re-visit after the load is complete.

**Comment by ianco** [comment]
I've duplicated this locally so am fixing the "500 server error"

**Comment by WadeBarnes** [comment]
> Other examples of this issue:
> 
> * https://orgbook-test.pathfinder.gov.bc.ca/en/organization/registration.registries.ca/BC0499790
> * https://orgbook-test.pathfinder.gov.bc.ca/en/organization/registration.registries.ca/BC0560364
> * https://orgbook-test.pathfinder.gov.bc.ca/en/organization/registration.registries.ca/BC0327946
> * https://orgbook-test.pathfinder.gov.bc.ca/en/organization/registration.registries.ca/BC0472519

The errors with these records went away after the complete data load was finished.

**Comment by WadeBarnes** [comment]
Resolved in 25c1c3a0b961ab4a0b82bd18b0ce0778a1d9bb18

Ensured FM0149073 still does not exist:
![image](https://user-images.githubusercontent.com/10504350/81073913-2a5d7900-8e9d-11ea-87c0-428911149898.png)

Viewed page for BC0131809:
![image](https://user-images.githubusercontent.com/10504350/81074008-46611a80-8e9d-11ea-8d82-8590cd281454.png)

- No error message.
- Available data is displayed.
